docs/deploy.md

```diff
@@ -0,0 +1,148 @@
+# Deploy Guide
+
+## Prerequisite Software
+
+The following software should be installed on the host before attempting to deploy Classy.
+
+- Docker
```

Comment:
@winstan Are there any other specific bits of information you'd like to see in the classy deploy instructions?  (just this block for deploy.md)

Reply:
I'd like to have a discussion on the system architecture of classy, after which I think the deploy instructions would be relatively obvious.
Simply, the ideal would be to separate out everything needing to be done as root (I think creating users/directories, setting permissions/group memberships) from everything else. The idea would be to separate what is done by a sysadmin and what is done by a classy administrator. Furthermore, classy administrators should not need sudo for anything.
You should also talk to Mike about certbot stuff. A sysadmin can most likely do that part for you.

I probably need to deploy this thing on a host myself to work out any issues in your deployment. Let me know when it would be a good time to do that.